src/LiquidityPoolConverter.ts

```diff
@@ -204,12 +205,12 @@ class IConversionEvent {
 }
 
 function handleConversion(event: IConversionEvent): void {
-  const fromAmount = decimalize(event.fromAmount, event.fromToken)
-  const toAmount = decimalize(event.toAmount, event.toToken)
-  const conversionFee = decimalize(event.conversionFee, event.toToken)
-  const protocolFee = decimalize(event.protocolFee, event.toToken)
+  const toToken = Token.load(event.toToken.toHexString())
+  const fromAmount = decimal.fromBigInt(event.fromAmount, DEFAULT_DECIMALS)
+  const toAmount = decimal.fromBigInt(event.toAmount, DEFAULT_DECIMALS)
```

Comment:
shouldn't we use toToken decimals here?

Reply:
Same as comment about fromToken. I know toToken Token entity is actually loaded here, but I wanted to use default decimals in both or neither to keep them consistent

Reply:
how is it consistent if the decimals are different and potentially the calculation is wrong?